docs/core/howto/ssl.rst

```diff
@@ -1,8 +1,3 @@
-
-:LastChangedDate: $LastChangedDate$
-:LastChangedRevision: $LastChangedRevision$
-:LastChangedBy: $LastChangedBy$
-
```

Comment:
My understanding is that these turds are unnecessary artifacts of the Lore migration that provide no utility, so when editing docs we remove them.  Irrelevant to this specific change, of course.